src/transformation/visitors/binary-expression/assignments.ts

```diff
@@ -179,7 +190,9 @@ const canBeTransformedToLuaAssignmentStatement = (
         }
 
         if (ts.isPropertyAccessExpression(element) || ts.isElementAccessExpression(element)) {
-            return true;
```

Comment:
This is a pretty drastic change, was this incorrect before, or what changed for this to get the opposite value now?

Reply:
See #1127